feature-libs/order/styles/components/order-overview/_order-overview.scss

```diff
@@ -22,10 +22,8 @@
         flex-grow: 1;
 
         @include media-breakpoint-up(lg) {
-          @include forVersion(1, 6.5) {
-            border-inline-end: 1px solid var(--cx-color-text);
-            margin-inline-end: 10rem;
-          }
+          border-inline-end: 1px solid var(--cx-color-text);
```

Comment:
forVersion(1, 6.5) means, styles are applied inclusively between versions 1 and 6.5.  do you still need these two lines?

Reply:
I'll check with Florent about this one. IF the intended use is just for 1, 6.5 to hide the payment info

Reply:
its removed now